Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -0,0 +1,1658 @@
+    command_results = CommandResults(
+        outputs_prefix='PrismaCloud.Account',
+        outputs_key_field='accountId',
+        readable_output=tableToMarkdown('Accounts Owner Details:',
+                                        responses,
+                                        removeNull=True,
+                                        headerTransform=pascalToSpace),
+        outputs=responses,
+        raw_response=responses
+    )
+    return command_results
+
+
+def host_finding_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    rrn = args.get('rrn')
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    all_results = argToBoolean(args.get('all_results', 'false'))
+
+    finding_types = argToList(args.get('finding_types'))
+    finding_types = [finding_type.lower() for finding_type in finding_types]
+    validate_array_arg(finding_types, 'Finding types', FINDING_TYPES_OPTIONS)
+
+    risk_factors = argToList(args.get('risk_factors'))
+    risk_factors = [risk_factor.upper() for risk_factor in risk_factors]
+    validate_array_arg(risk_factors, 'Risk factors', RISK_FACTORS_OPTIONS)
+
+    response = client.host_finding_list_request(str(rrn), finding_types, risk_factors)
+    total_response_amount = len(response)
+    if not all_results and limit and response:
+        response = response[:limit]
+    for response_item in response:
+        change_timestamp_to_datestring_in_dict(response_item)
+    readable_responses = deepcopy(response)
+
+    nested_headers = {'sourceData.accountId': 'Source Data Account ID',
+                      'sourceData.arn': 'ARN',
+                      }
+    headers = ['accountId', 'regionId', 'findingId', 'type', 'source', 'severity', 'status', 'createdOn', 'updatedOn',
+               'normalizedNames', 'scanId', 'resourceCloudId', 'Source Data Account ID', 'ARN', 'title', 'description',
+               'resourceUrl']
+    for readable_response in readable_responses:
+        extract_nested_values(readable_response, nested_headers)
+
+    command_results = CommandResults(
+        outputs_prefix='PrismaCloud.HostFinding',
+        outputs_key_field='findingId',
+        readable_output=f'Showing {len(readable_responses)} of {total_response_amount} results:\n'
+                        + tableToMarkdown('Host Finding Details:',
+                                          readable_responses,
+                                          headers=headers,
+                                          removeNull=True,
+                                          url_keys=['resourceUrl'],
+                                          headerTransform=pascalToSpace),
+        outputs=response,
+        raw_response=response
+    )
+    return command_results
+
+
+def permission_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    user_id = args.get('user_id')
+    query = args.get('query')
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT)) or int(DEFAULT_LIMIT)
```

Comment:
It will not, added it so lint won't shout
![image](https://user-images.githubusercontent.com/72088126/221796522-2c6dd5a5-6523-4c23-ba6b-17351f33c6b8.png)